src/server/routes/reviews.ts

```diff
@@ -45,7 +45,7 @@ router.post('/:entityType/:bbid/reviews', auth.isAuthenticated, auth.isAuthorize
 		review
 	);
 
-	let newAccessToken = '';
+	let newAccessToken: any = '';
```

Comment:
Should this be `string` rather than `any`?

Reply:
It shouldn't be string, because it returns a JSON object and we're using one of its values below: `newAccessToken.access_token`.

I suppose we can do something like this:
`let newAccessToken: Record<string, any> = {};`

What do you think?

Reply:
Either that or not assigning an initial value at all:
```suggestion
	let newAccessToken: any;
```
If the token fetching logic throws, we don't even reach the code block where we try to access `newAccessToken.access_token`, so it does not matter whether `newAccessToken` is an object.

Reply:
I just realised that we already have a type defined for this on `bookbrainz-data`. I think this works.